qiskit_ibm_runtime/utils/estimator_pub_result.py

```diff
@@ -0,0 +1,130 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+========================================================================================
+EstimatorPubResult result classes (:mod:`qiskit_ibm_runtime.utils.estimator_pub_result`)
+========================================================================================
+
+.. autosummary::
+   :toctree: ../stubs/
+"""
+
+from __future__ import annotations
+
+from typing import Sequence, TYPE_CHECKING
+
+from qiskit.primitives.containers import PubResult
+
+if TYPE_CHECKING:
+    from plotly.graph_objects import Figure as PlotlyFigure
+
+
+class EstimatorPubResult(PubResult):
+    """Result of Estimator Pub."""
+
+    def draw_zne_evs(
+        self,
+        indices: Sequence[tuple[int, ...]] | None = None,
+        names: Sequence[str] | None = None,
+        n_stds: int = 1,
+        max_mag: float = 10,
+        max_std: float = 0.2,
+        height: int = 500,
+        width: int = 1000,
+        n_cols: int = 4,
+        colorscale: str = "Aggrnyl",
+    ) -> PlotlyFigure:
```

Comment:
<this comment applies to all functions/methods>

Nit: the prevalent style in `qiskit` is the full `num_` prefix

```suggestion
    def draw_zne_evs(
        self,
        indices: Sequence[tuple[int, ...]] | None = None,
        names: Sequence[str] | None = None,
        num_stds: float = 1.0,
        max_mag: float = 10,
        max_std: float = 0.2,
        height: int = 500,
        width: int = 1000,
        num_cols: int = 4,
        colorscale: str = "Aggrnyl",
    ) -> PlotlyFigure:
```